test/test.js

```diff
@@ -49,15 +62,15 @@ suite('at-driver', () => {
     return Promise.race([whenClosed, connect(6543, SUB_PROTOCOL)]);
   });
 
-  test('rejects unspecified protocol', async () => {
-    const {whenClosed} = await run([]);
-    return Promise.race([whenClosed, invert(connect(4382))]);
-  });
+  // test('rejects unspecified protocol', async () => {
```

Comment:
@jugglinmike I might be completely missing the relevant point in the spec, but not sure this is a check we need anymore with sub-protocols

Reply:
Should we use `test.skip`?

Reply:
I chose to comment because we should either re-enable it because something about it should be tested or remove it entirely.  Just wasn't sure which without asking

Reply:
Thanks, @gnarf. Although AT Driver is a ways off from being a standard, I think we're past the point where explicit versioning is helpful. I'll remove these tests.